### DISCUSSION TITLE
Can you build it for cutefishos

### DISCUSSION THREAD

**Discussion by sabhiron** [kraxarn/spotify-qt#195] [Ideas]
Hey kraxie 
I am Abhishek I need the help of your can you build Spotify qt-quick for cutefish OS
or if you do not can I use your source code to build it myself

**Comment by kraxarn** [comment]
As CutefishOS seems to be built off Ubuntu, they should have the same build instructions and simply running `sudo apt install build-essential qt5-default libqt5svg5-dev` before the [instructions provided in the readme](https://github.com/kraxarn/spotify-qt#building-yourself) should allow you to build it yourself if you don't want to use the provided binaries.

In case you meant [spotify-qt-quick](https://github.com/kraxarn/spotify-qt-quick) and just opened the discussion in the wrong repo, try installing Qt using the previously mentioned command, and then follow the instructions [from the readme page](https://github.com/kraxarn/spotify-qt-quick#building-yourself). Binaries for it are currently not provided, as it's still early in development.
